target-platform/p2-repo-common/pom.xml

```diff
@@ -82,6 +82,11 @@
 									<version>${org.eclipse.kura.sun.misc.version}</version>
 								</artifactItem>
 								<artifactItem>
+								    <groupId>org.json</groupId>
```

Comment:
@ctron As the code base is really big for the Wires Web Integration, I would first concentrate on integrating it and then changing or modifying it. As this would be in different release, I believe we would have no problem in using JSON for now and if all the committers agree on having GSON for more flexibilities, I would have no problem in changing that later on.

So, for now, we can keep it and let the interested users play with Wires and then we can discuss on the JSON or GSON library integration.